.github/workflows/check_signoffs.sh

```diff
@@ -0,0 +1,38 @@
+# Check commits in this pull request for Signed-off-by trailers, and assure that
+# the committer has given his signoff.
+set -e
+
+# Get the base branch reference from the PR
+REMOTE=${REMOTE:-origin}
+BASE_BRANCH=${BASE_BRANCH:-master}
+
+# Fetch the base branch
+git fetch $REMOTE $BASE_BRANCH
+
+# Get the list of commits in the pull request against the base branch
+commits=$(git rev-list --reverse $REMOTE/$BASE_BRANCH..HEAD)
+
+# Check each commit for Signed-off-by trailer
+test_signed_off_ok=true
+for commit in $commits; do
+	
+	signoffs=$(git show --format="%(trailers:key=Signed-off-by,valueonly=true)" -s $commit)
+	#echo -e "\tsignoffs=${signoffs}"
```

Comment:
Was this for debugging? Should it be removed now? Same below.

Reply:
Since this is just a piece of infrastructure code, I left these debugging statements in to marginally speed up debugging a PR pipeline failure. I guess I can remove them, if you really object.

Reply:
No objection. Just looking at things from a consistency perspective. I would feel strongly if this were a shipping script.